kili/cli.py

```diff
@@ -66,11 +68,51 @@ def list_project(api_key: str,
           showindex=False, colalign=("left", "left", "right", "left")))
 
 
+@project.command(name='create')
+@click.option('--api-key', type=str, envvar='KILI_API_KEY', required=True,
+              help=(
+                  'Your Kili API key (overrides the KILI_API_KEY environment variable). '
+                  'If not passed, requires the KILI_API_KEY environment variable to be set.'
+              ))
+@click.option('--endpoint',
+              type=str,
+              default='https://cloud.kili-technology.com/api/label/v2/graphql',
+              help='Your Api Enpoint')
+@click.option('--interface', type=click.Path(exists=True), required=True,
+              help='Path pointing to your json interface file')
+@click.option('--title', type=str, required=True,
+              help='Project Title')
+@click.option('--input-type', type=click.Choice(INPUT_TYPE), required=True,
+              help='Project input data type')
+@click.option('--description', type=str, default='',
+              help='Project description')
+# pylint: disable=too-many-arguments
+def create_project(api_key: str,
+                   endpoint: str,
+                   input_type,
+                   interface: str,
+                   title: str,
+                   description: str):
+    """Create a Kili project.
+    Examples:
+        $ kili project create --project-id <project_id> \
+        --interface path/to/interface.json --input-type TEXT --title "Invoice annotation project"
```

Comment:
add a tab here

Reply:
Yeah, this is taken in the documentation ticket that is next to come ;)